src/app/utils/externalLinks.ts

```diff
@@ -48,6 +48,7 @@ export const faucet = {
   [Layer.consensus]: faucetUrl,
   [Layer.emerald]: `${faucetParaTimeBaseUrl}emerald`,
   [Layer.sapphire]: `${faucetParaTimeBaseUrl}sapphire`,
+  [Layer.pontusx]: `${faucetParaTimeBaseUrl}pontusx`,
```

Comment:
Do we support faucet for 3rd parties?

Reply:
if we don't know that yet I am not expecting we will get answers soon. Let's revisit in the future.

Reply:
> Do we support faucet for 3rd parties?

Now we know that the right way to get test tokens is to send an email. Updated the URL to a `mailto:` link, so the button now opens the email client. This is the best we can do for now.